src/mapdata.cpp

```diff
@@ -1301,6 +1301,7 @@ void map_data_common_t::load( const JsonObject &jo, const std::string & )
         JsonObject data = jo.get_object( "examine_action" );
         examine_actor = iexamine_actor_from_jsobj( data );
         examine_actor->load( data );
+        examine_func = iexamine_functions_from_string( "none" );
```

Comment:
I think it is error prone and wrong to give something both an examine func and a examine actor, because it means we can potentially accidentally ignore the actor for the terrain/furniture - I think a hard failure is better than a silent one.

Reply:
Well, examine_func is a reference not a pointer, so I don't think I can just set it to NULL - it's this, some other function, or undefined behavior.

Reply:
I don't suppose one could create an interface function, so that trying to invoke its examine_func would invoke the examine_actor instead? I haven't looked at the code in question, admittedly.

Reply:
Or we go the simple route and make an iexamine that complains loudly about being used. I'd recommend that.

Reply:
57bb662